irony-cdb-clang-complete.el

```diff
@@ -40,8 +40,12 @@
 
 (defun irony-cdb-clang-complete--locate-db ()
   (when buffer-file-name
-    (irony--awhen (locate-dominating-file buffer-file-name ".clang_complete")
-      (concat (file-name-as-directory it) ".clang_complete"))))
+    (irony--awhen
```

Comment:
I'm not sure I agree on adding this feature to `.clang_complete`. Unless proven otherwise, a `.clang_complete` should be found in a parent directory of the source file. Let's keep it that way for now.